src/components/Plugin/DaoModule/CustomBlock.vue

```diff
@@ -111,6 +136,40 @@ export default {
         default:
           return false;
       }
+    },
+    approvalData() {
+      if (this.questionDetails.currentBond === '0.0') {
+        return {
+          timeLeft: 'Time to finish: -',
+          decision: 'Current final answer: -'
+        };
+      }
+
+      const time = format(this.questionDetails.endTime * 1e3);
+      if (this.questionDetails.finalizedAt) {
+        const timeLeft = `Finished: ${time}`;
+        if (this.questionDetails.isApproved) {
+          return {
+            decision: 'Final Answer: Yes',
```

Comment:
Something i forgot to say is that across the site we use "Sentence case" and not "Title Case". It's just a small details but ideally it should be the same for plugins
```suggestion
            decision: 'Final answer: Yes',
```